v2.0/app/utils/messenger.js

```diff
@@ -27,7 +27,12 @@ export async function apiCall(
   const formData = new FormData();
   Object.keys(data).map(k => (formData.append(k, data[k])));
 
-  const response = await fetch(`${API_HOST}/v3${destinationUrl}`, {
+  // make leading '/' optional
+  if (destinationUrl.charAt(0) === '/') {
```

Comment:
If destination is null the whole site will crash.
To be safe do destinationUrl && ...

Reply:
Added protection against null or 0 length destinationUrl